### PR TITLE
Adds simple initial skeleton

### PR DESCRIPTION
**Pull request by Tankenstein**
It's got:
- Typescript
- Postgres with migrations and a query builder
- Prettier auto-formatting
- Jest for tests
- Express with some boilerplate to support `async` properly
- CI

It's ready to be deployed to heroku.

It's missing:
- Linting, if we want it
- Structure, if we want it atm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "*.ts": [
+      "npm run format"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cov-clear/backend.git"
+  },
+  "author": "",
+  "license": "UNLICENSED",
```
Comment (igalarzab) [inline package.json]: Apache?

File: src/database/migrations/V1__addUsers.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Knex from 'knex';
+
+export async function up(db: Knex) {
+  await db.schema.createTable('example_users', (table) => {
+    table.bigIncrements('id');
+    table.timestamps(true, true);
+    table.string('email').unique().notNullable();
+    table.string('hashed_password').notNullable();
```
Comment (igalarzab) [inline src/database/migrations/V1__addUsers.ts]: I guess this is an example right? We are not having passwords

File: package.json (hunk 1 above)
Comment (Tankenstein) [reply package.json]: Will add yep, left it as a todo until we decide.

File: src/database/migrations/V1__addUsers.ts (hunk 2 above)
Comment (Tankenstein) [reply src/database/migrations/V1__addUsers.ts]: Yes, it's an example for creating migrations.

**Pull request merged**